exe/ruby-lsp

```diff
@@ -20,7 +20,39 @@ end
 
 require_relative "../lib/ruby_lsp/internal"
 
-if ARGV.include?("--debug")
+require "optparse"
+
+options = {}
+parser = OptionParser.new do |opts|
+  opts.banner = "Usage: ruby-lsp [options]"
+
+  opts.on("--version", "Print ruby-lsp version") do
+    puts RubyLsp::VERSION
+    exit(0)
+  end
+
+  opts.on("--debug", "Launch the Ruby LSP with a debugger attached") do
+    options[:debug] = true
+  end
+
+  opts.on("-h", "--help", "Print this help") do
+    puts opts.help
+    puts
+    puts "See https://shopify.github.io/ruby-lsp/ for more information"
+    exit(0)
+  end
+end
+
+options = begin
+  parser.parse!
+rescue OptionParser::InvalidOption => e
+  warn(e)
+  warn("")
+  warn(parser.help)
+  exit(1)
+end
+
+if options[:debug]
```

Comment:
Is this `if` statement missing an `end`?

Reply:
Oh, yeah, indeed. The other `if` statement was removed. Why is the integration test failing then 🤔.